src/Microsoft.Performance.SDK.Runtime.NetCoreApp/Plugins/IPluginsConsumer.cs

```diff
@@ -8,25 +8,25 @@ namespace Microsoft.Performance.SDK.Runtime.NetCoreApp.Plugins
     public interface IPluginsConsumer
```

Comment:
>IPluginsConsumer [](http://example.com/codeflow?start=21&length=16)

I don't think the consumer should know that the ProcessSource came from a plugin - they don't need to know this. They really only need to know that a ProcessingSource was discovered. To this end, we might want to rename this to IProcessingSourceConsumer. Thoughts?

Reply:
This interface is part of the plugin management system I started creating. Consumers will eventually need to know what plugin a processing source came from; that information (if it can be determined) is sent by the plugins loader to these consumers.

Right now the name doesn't make too much sense since the management system halted development, but one day it will 😆

Reply:
ah, I didn't look closely enough at the other parameters. makes sense.

Reply:
I think this plays into #22 that Luke mentioned elsewhere. In this specific case, I think ProcessingSourceConsumer makes sense given what it does.